text/0000-process-stdio-redirection.md

```diff
@@ -0,0 +1,161 @@
+- Feature Name: process-stdio-redirection
+- Start Date: 2015-04-10
+- RFC PR:
+- Rust Issue:
+
+# Summary
+
+Update the standard library API with the ability to redirect stdio of child
+processes to any opened file handle or equivalent.
+
+# Motivation
+
+The current API in `std::process` allows to either pipe stdio between parent and
```

Comment:
I think that this motivation section may want to be updated with the current state of the standard library because with the `FromRaw{Fd,Handle}` implementations on `Stdio` it's actually possible to do this. It looks like this RFC gets us to a point of a slightly-more-ergonomic version of what we have today, but I think it would be worth it to explore the motivation to make sure the gains are worth the additions.